i18n/miraheze/en.json

```diff
@@ -95,30 +95,28 @@
 	"grouppage-steward": "m:Stewards",
 	"group-sysadmin": "System administrators",
 	"group-sysadmin-member": "{{GENDER:$1|system administrator}}",
-	"grouppage-sysadmin": "m:System administrators",
+	"grouppage-sysadmin": "m:Tech:SRE Volunteers",
 	"group-interwiki-admin": "Interwiki Administrator",
 	"group-interwiki-admin-member": "interwiki administrator",
 	"grouppage-interwiki-admin": "m:Interwiki administrators",
```

Comment:
Special:MyLanguage?

Reply:
Which file is this in? I think I changed this?

Reply:
> i18n/miraheze/en.json

Reply:
I'm so confused, some groups I'm changing in /overrides and others in /miraheze. But /miraheze is not using the localised messages and Special:MyLanguage, so what needs changing?